.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 python:
   - "2.7"
```

Comment:
Do we want to drop testing for py2 as well and add py37?

Reply:
Good point.  Missed that reference.  See a38a4cbbb2149690ff4b80cb0da1bbfd5e713806.